src/main/webapp/view/discussion/discussion.js

```diff
@@ -21,17 +23,27 @@ export const COMMENT_TYPES = {
   [COMMENT_TYPE_REPLY]: {
     name: 'Reply',
     badgeStyles: ['badge-secondary', 'float-right', 'badge', 'badge-pill'],
+    hasMarkAs: false,
```

Comment:
hasMarkAs is very vague and isn't clear what is means. Would it be better if it was called selectableType.